v1/definition.yaml

```diff
@@ -109,23 +109,25 @@ paths:
                 content-type: '{{storage.headers.content-type}}'
                 etag: '{{storage.headers.etag}}'
                 cache-control: '{{options.response_cache_control}}'
+                content-language: '{{storage.headers.content-language}}'
+                vary: '{{storage.headers.vary}}'
```

Comment:
Don't we want to store the headers too? L98 doesn't set them for storage.

Reply:
L98 is the GET request to the storage.